examples/model_selection/plot_grid_search_stats.py

```diff
@@ -0,0 +1,548 @@
+# our beliefs of how the means are distributed before looking at the data,
+# and multiply it by a likelihood function that computes how likely our
+# observed mean differences are, given the values that the mean of differences
+# could take.
+#
+# Bayesian estimation can be carried out in many forms to answer our question,
+# but in this example we will implement the approach suggested by Benavoli and
+# collegues [4]_.
+#
+# One way of defining our posterior using a `closed-form expression
+# <https://en.wikipedia.org/wiki/Closed-form_expression>`_ is to select a
+# prior `conjugate <https://en.wikipedia.org/wiki/Conjugate_prior>`_ to the
+# likelihood function. Benavoli and collegues [4]_ show that when comparing the
+# performance of two classifiers we can model the prior as a Normal-Gamma
+# distribution (with both mean and variance unknown)
+# `conjugate <https://en.wikipedia.org/wiki/Conjugate_prior>`_ to a normal
+# likelihood, to thus express the posterior as a normal distribution.
+# Marginalizing out the variance from this normal posterior, we can define the
+# posterior of the mean parameter as a Student T distribution. Specifically:
+#
+# .. math::
+#    St(\mu;n-1,\overline{x},(\frac{1}{n}+\frac{n_2}{n_1})\hat{\sigma}^2)
+#
+# where :math:`n` is the total number of observations.
+#
+# Notice that we are using Nadeau and Bengio's corrected variance in our
+# Bayesian approach as well.
+#
+# Let's compute the posterior:
+
+# intitialize random variable
+t_post = t(
+    df, loc=np.mean(differences),
+    scale=correct_std(differences, n, n_train, n_test)
+)
+
+# %%
+# Let's plot the posterior distribution:
+
+x = np.linspace(t_post.ppf(0.001), t_post.ppf(0.999), 100)
+
+plt.plot(x, t_post.pdf(x))
+plt.xticks(np.arange(-0.04, 0.06, 0.01))
+plt.fill_between(x, t_post.pdf(x), 0, facecolor='blue', alpha=.2)
+plt.ylabel("Probability density")
+plt.xlabel("Mean difference")
+plt.title("Posterior distribution")
+plt.show()
+
+# %%
+# We can calculate the probability that the first model is better than the
+# second by computing the area under the curve of the posterior distirbution
+# from zero to infinity. And also the reverse: we can calculate the probability
+# that the second model is better than the first by computing the area under
+# the curve from minus infinity to zero.
+
+better_prob = t_post.sf(0)
+
+print(f"Probability of {model_scores.index[0]} being more accurate than "
+      f"{model_scores.index[1]}: {better_prob:.3f}")
+print(f"Probability of {model_scores.index[1]} being more accurate than "
+      f"{model_scores.index[0]}: {1 - better_prob:.3f}")
+
+# %%
+# In contrast with the frequentist approach, we can compute the probability
+# that one model is better than the other.
+#
+# Note that we obtained similar results as those in the frequentist approach.
+# Given our choice of priors we are in essence performing the same
+# computations, but we are allowed to make different assertions.
+
+# %%
```

Comment:
```suggestion
# %%
Region of Practical Equivalence
-------------------------------
```
This would give more structure and therefore help reading, in my opinion.